### PR TITLE
Set KUBECONFIG every time within the container

### PR DESCRIPTION
**Pull request by sdake**

### CODE REVIEW

**Comment by ericvn** [comment]
Assume this is because it's getting passed in from the host. Does it make sense to always set it or should be not pass in the value from the host which should leave it unset?

**Comment by sdake** [comment]
@ericvn As a followup to https://github.com/istio/tools/pull/899 and https://github.com/istio/tools/pull/900, Take whatever the host has as a KUBECONFIG file in $HOME/.kube/config, and use that for the container.

In future PRs, I would like to add:
 - the ability to handle `KUBECONFIG=/tmp/.kube/config make test`
- the ability for kind to be unaffected by any environmental changes.
Cheers
-steve

File: common/scripts/setup_env.sh [hunk 1]
```diff
@@ -78,6 +78,8 @@ export TARGET_OUT_LINUX="${TARGET_OUT_LINUX:-$(pwd)/out/linux_amd64}"
 export CONTAINER_TARGET_OUT="${CONTAINER_TARGET_OUT:-/work/out/${TARGET_OS}_${TARGET_ARCH}}"
 export CONTAINER_TARGET_OUT_LINUX="${CONTAINER_TARGET_OUT_LINUX:-/work/out/linux_amd64}"
 
+export KUBECONFIG="/home/.kube/config"
```
Comment (brian-avery) [inline common/scripts/setup_env.sh:81]: Should this be $HOME?

Comment (sdake) [reply common/scripts/setup_env.sh:81]: The flow of .kube/config file into the build container is as follows

1. run make test
1. make, as part of its execution creates and runs a build container
1. the build container, as part of its execution, runs this script
1. the build container then executes the job

Inside the build container, KUBECONFIG is always `/home/.kube/config`

Atleast with, https://github.com/istio/tools/pull/899

Comment (sdake) [reply common/scripts/setup_env.sh:81]: job^target

**Pull request merged**